### DISCUSSION TITLE
How to save a copy of outbound emails?

### DISCUSSION THREAD

**Discussion by bluepuma77** [postalserver/postal#2628] [Help with using Postal]
We are currently evaluating using postal for sending transactional emails via API. We have operations people that love to look into the sent emails folder to check that everything works fine. So they want a copy of the outgoing emails in Sent. Is that possible?

**Comment by willpower232** [comment]
Postal will retain the contents of emails for 30 days by default and the meta information for 60 days.

It is probably better to keep an archive in whatever software you're using to send emails. That way you can attach all the extra context you need.

Depending on your programming language, this may be built into the API library you use.
